_posts/2019-09-06-snowplow-r116-madara-rider/index.md

```diff
@@ -0,0 +1,136 @@
+
+Which domain will be used in the `Set-Cookie` header is determined by matching the domains from the `Origin` header of the request to the specified list. The first match is used. If no matches are found, the fallback domain will be used, if configured. If no `fallbackDomain` is configured, the cookie will be tied to the full collector domain.
+
+If you specify a main domain in the list, all subdomains on it will be matched. If you specify a subdomain, only that subdomain will be matched. For example:
+- `example.com` will match `Origin` headers like `example.com`, `www.example.com` and `secure.client.example.com`
+- `client.example.com` will match an `Origin` header like `secure.client.example.com` but not `example.com` or `www.example.com`.
+
+(We've written about the impact of ITP2.1 before: [here][itp-how] and [here][itp-why].)
+
+<h2 id="paths">2. Custom request paths</h2>
+
+We have been aware for some time that the Snowplow JavaScript tracker has been targeted by some ad-blocking providers, whose software blocks traffic to the paths used by the HTTP requests the tracker sends to the collector.
+
+We firmly believe that data subjects's rights must be protected and we are providing [tools and advice][gdpr] that makes that easy for Snowplow users. While we understand how ad-blockers might see a Snowplow tracker as a privacy invasion, we believe that is misguided. Snowplow enables event tracking and as such data flows through pipelines we've built. But we have no access to that data and no interest in it beyond ensuring that it is safely collected and stored in our users's own data warehouses. Blocking the Snowplow tracker is in effect preventing website owners from understanding their users and improving their service.
+
+The request paths we use have traditionally been intrinsically tied to how the collector (and downstream processes) work. With version `0.16.0` we are giving users of the Scala Stream Collector the option to map their own custom paths to the ones used under the hood. That way, users will be able, for example, to send requests to a path like `/sp/track`.
+
+The path can be changed when initialising the JavaScript tracker, as explained [here][post-path]. See below in the <a href="upgrading">Upgrading</a> section for details on how to configure the mappings in the collector.
+
+<h2 id="cache">3. Cache-Control headers</h2>
+
+To increase the likelihood that the Snowplow pixel will not be cached by a browser or proxy, we have added the following directives to the `Cache-Control` header of the collector's response: `no-cache`, `no-store`, `must-revalidate`.
+
+<h2 id="eer">4. EmrEtlRunner</h2>
+
```

Comment:
Is there a missing section here?

Reply:
Ah - just seen the conversation above - if we can omit that section let's remove and then we're good to publish - thanks again